examples/pump-app/pump-common/gen/CHIPClusters.cpp

```diff
@@ -81,6 +84,23 @@ namespace Controller {
 // TODO(#4503): length should be passed to commands when byte string is in argument list.
 // TODO(#4503): Commands should take group id as an argument.
 
+namespace {
+
+// The helper function to encode a single scalar (numeric / string / bytearray) attributes.
```

Comment:
could we take this into a library function?

'gen' code should depend on generated input data, where as this method does not seem to depend on any input code generation.

Reply:
Moved to member function of WriteClientHandle.